css-two-column-layout/styles.css

```diff
@@ -0,0 +1,74 @@
+.container {
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 910px;
+}
+
+.row {
+  justify-content: center;
+  display: flex;
+}
+
+.inner-row {
+  display: flex;
+}
+
+button {
+  border-color: #d14c45;
+  background-color: white;
+  border-width: 0.13rem;
+  color: #d14c45;
+  font-weight: bold;
+  padding-left: 2.6rem;
+  padding-right: 2.6rem;
+  padding-top: 0.6rem;
+  padding-bottom: 0.6rem;
+}
+
+.image {
+  width: 25%;
+}
+
+img {
+  width: 100%;
+}
+
+.text {
+  width: 70%;
+  margin-top: 0.85rem;
+}
+
+.header-text {
+  margin-bottom: -30px;
```

Comment:
Negative margins should generally be avoided. If you find yourself needing a negative margin, it is usually an indicator that a sibling element just needs to have its margin reduced.